README.md

```diff
@@ -36,12 +50,33 @@ jobs:
       # ===========================================================================
       - name: Derive appropriate SHAs for base and head for `nx affected` commands
         id: setSHAs
-        uses: nrwl/nx-set-shas@v1
+        uses: nrwl/nx-set-shas@v2
+        with:
+          set-environment-variables-for-job: 'false'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     
       - run: |
           echo "BASE: ${{ steps.setSHAs.outputs.base }}"
           echo "HEAD: ${{ steps.setSHAs.outputs.head }}"
 
+      # ===========================================================================
+      # OPTION 3) Specify all parameters
```

Comment:
This is not really what was intended by OPTIONs here.

There are two fundamentally different ways of using the SHAs - via environment variables, or via github job outputs.

This is not a third OPTION when it comes to fundamental usage of the SHAS, so please remove.

The configuration options section below was intended to exhaustively cover all the options available